src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validationCase } = require('./validationCase');
+
+const server = () => http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  const params = new URLSearchParams(url.search);
+  const toCase = params.get('toCase');
+  const textToConvert = url.pathname.slice(1);
+
+  function errorRes(errorMessages) {
+    return {
+      errors: errorMessages
+        .map(errorMessage => ({ message: errorMessage })),
+    };
+  }
+
+  function successRes(
+    originalCase,
+    targetCase,
+    originalText,
+    convertedText,
+  ) {
+    return {
+      originalCase,
+      targetCase,
+      originalText,
+      convertedText,
+    };
+  };
```

Comment:
As i think, this helps to ensure that the response will have valid structure, but it's too redundant, you can just create this object in-place.
```suggestion
```